src/commands/createNewProject/ProjectCreateStep/ScriptProjectCreateStep.ts

```diff
@@ -106,6 +106,9 @@ local.settings.json
 # TypeScript output
 dist
 out
+
+# Managed dependencies folder
+ManagedDependencies/
```

Comment:
This should also go in the `.funcignore` file - which applies when deploying

Reply:
We no longer need to make this change. Please see above comment.